docs/pages/kcardgrid.vue

```diff
@@ -0,0 +1,563 @@
+          />
+        </KCardGrid>
+      </DocsShow>
+
+      <!-- eslint-disable -->
+      <DocsShowCode language="html">
+        <KCardGrid layout="1-2-2">
+          <KCard
+            v-for="i in 3"
+            ...
+          />
+        </KCardGrid>
+      </DocsShowCode>
+      <!-- eslint-enable -->
+
+      <h4>
+        '1-2-3' grid
+        <DocsAnchorTarget anchor="#1-2-3-grid" />
+      </h4>
+      <p>Displays a grid with 1 card per row on smaller screens, 2 cards per row on medium screens, and 3 cards per row on larger screens. <DocsToggleButton contentId="more-1-2-3-grid" showText="Show full definition" hideText="Hide full definition" /></p>
+
+      <DocsToggleContent id="more-1-2-3-grid">
+        <DocsTable>
+          <tr>
+            <th></th>
+            <th>Level 0</th>
+            <th>Level 1</th>
+            <th>Level 2</th>
+            <th>Level 3</th>
+            <th>Level 4</th>
+            <th>Level 5</th>
+            <th>Level 6</th>
+            <th>Level 7</th>
+          </tr>
+          <tr>
+            <th>Cards per row</th>
+            <td>1</td>
+            <td>1</td>
+            <td>2</td>
+            <td>2</td>
+            <td>3</td>
+            <td>3</td>
+            <td>3</td>
+            <td>3</td>
+          </tr>
+        </DocsTable>
+      </DocsToggleContent>
+
+      <DocsShow block>
+        <KCardGrid layout="1-2-3">
+          <DocsKCard
+            v-for="i in 5"
+            :key="i"
+            :headingLevel="5"
+            :prependTitle="`(${i})`"
+          />
+        </KCardGrid>
+      </DocsShow>
+
+      <!-- eslint-disable -->
+      <DocsShowCode language="html">
+        <KCardGrid layout="1-2-3">
+          <KCard
+            v-for="i in 5"
+            ...
+          />
+        </KCardGrid>
+      </DocsShowCode>
+      <!-- eslint-enable -->
+
+      <h3>
+        Layout customization
+        <DocsAnchorTarget anchor="#layout-customization" />
+      </h3>
+
+      <p>Base layouts can be customized or even completely overriden. Refer to <code>useKCardGrid</code> (TBD).</p>
+
+      <h3>
+        Card height, content tolerance and  alignment
+        <DocsAnchorTarget anchor="#card-height-and-alignment" />
+      </h3>
+
+      <p>Cards displayed in <code>KCardGrid</code> stretch vertically to fit their content, making a grid row height match its tallest card.</p>
+
+      <p><em>Setting height on cards is discouraged. Instead, manage height bottom-up, for example by setting height on card sections, using text truncation, or other ways to limit its inner content.</em> Such approaches ensure content tolerance, prevent from unexpected overflows or excessive height, and keep vertical alignment of card sections consistent on a grid row. This is especially important when dealing with unknown lenghts or amounts of content displayed in cards. Consider:</p>
+
+      <DocsShow language="html">
+        <KCardGrid layout="1-2-3">
```

Comment:
Im not completely sure if this is part of the scope of this PR, but it seems that we have a small bug on responsiveness when we pass from 2 columns to just 1 column. I was wondering if this a KCardGrid thing or this was because of the docs not being responsive. But I saw that this empty space is controlled by KCardGrid so its the responsibility of KCardGrid.

https://github.com/user-attachments/assets/0cfdef90-fc40-4601-9346-308260ade057

Reply:
Yes, seems to be a bug with grid behavior. I will investigate.

Reply:
Done, it was likely caused by unpreciseness (perhaps some rounding errors) of `flex-basis`. Using `width` instead helped.